src/components/NFTPage/PreviewCard.vue

```diff
@@ -284,7 +284,14 @@ export default {
     handleClickViewContent() {
       this.$emit('view-content');
     },
-    handleClickViewContentUrls(type) {
+    handleClickViewContentUrls(e, contentUrl) {
+      const type = this.getContentUrlType(contentUrl);
```

Comment:
Besides `sameAs` field, do we need to handle if the content URL is a PDF too?

Reply:
UI only shows the "PDF" button if it's in sameAs
Also we dont really know if it's a pdf unless url explicitly end with .pdf
But we can add this for content url too if needed